### PR TITLE
[build]Build/battle/blurify

### PR DESCRIPTION
**Pull request by lina**

### CODE REVIEW

**Comment by codydaig** [comment]
Before this gets merged in, we should merge #11 in so you can rebase and make sure the user model tests pass with the ones that are currently marked as pending. Also, these should be squashed into one commit.

File: client/app/services/services.js [hunk 1]
```diff
@@ -110,6 +110,16 @@ angular.module('battlescript.services', [])
...
   }
 
+  var pointsChange = function(username, pointIncrease) {
+    console.log('Services pointsChange fired');
+    console.log(username, pointIncrease);
+    return $http({
+      method: 'POST',
+      url: '/api/users/pointschange',
+      data: {username: username, pointIncrease: pointIncrease}
+    })
```
Comment (codydaig) [inline client/app/services/services.js]: Semi colon?

File: client/app/services/services.js [hunk 2]
```diff
@@ -110,6 +110,16 @@ angular.module('battlescript.services', [])
...
+  }
```
Comment (codydaig) [inline client/app/services/services.js]: semi colon?

File: client/app/services/services.js [hunk 3]
```diff
@@ -121,7 +131,8 @@ angular.module('battlescript.services', [])
     getAuthUser: getAuthUser,
     getStats: getStats,
     statChange: statChange,
-    getLeaderboard: getLeaderboard
+    getLeaderboard: getLeaderboard,
+    pointsChange: pointsChange
   }
 
 })
```
Comment (codydaig) [inline client/app/services/services.js:144]: semi colon?

File: server/users/userController.js [hunk 4]
```diff
@@ -117,7 +117,8 @@ module.exports = {
         res.send({
           totalWins: user.totalWins,
           currentStreak: user.currentStreak,
-          longestStreak: user.longestStreak
+          longestStreak: user.longestStreak,
+          totalPoints: user.totalPoints
         })
       })
```
Comment (codydaig) [inline server/users/userController.js]: semi colon?

File: client/app/battle/battle.js [hunk 5]
```diff
@@ -1,6 +1,76 @@
...
+        $scope.getStats($scope.username);        
+      }
+    });
+
+  });
+
+
+
+
+
```
Comment (codydaig) [inline client/app/battle/battle.js:66]: Why all the whitespace?

File: server/users/userRoutes.js [hunk 6]
```diff
@@ -11,6 +11,8 @@ module.exports = function (app) {
   app.get('/stats', userController.stats);
   app.post('/statchange', userController.statChange);
   app.get('/leaderboard', userController.leaderboard);
+  app.post('/pointschange', userController.pointsChange);
+
```
Comment (codydaig) [inline server/users/userRoutes.js:18]: extra white space?

File: client/app/battle/battle.js [hunk 7]
```diff
@@ -181,6 +263,71 @@ angular.module('battlescript.battle', [])
...
+      } else {
+        $scope.userEditor.setValue(_.shuffle(textChunk1.split('')).join('') + textChunk2 + textChunk3);            
+      }
+    });
+  }
+
+
+
+
+
```
Comment (codydaig) [inline client/app/battle/battle.js:300]: white space?

**Comment by codydaig** [comment]
LGTM

**Comment by Kevbyte** [comment]
LGTM

**Comment by AndrewKishino** [comment]
LGTM

**Pull request merged**